definitions/subworkflows/pvacseq.cwl

```diff
@@ -30,8 +31,10 @@ inputs:
     expression_tool:
         type: string?
         default: 'kallisto'
-    alleles:
-        type: string[]
+    allele_file:
```

Comment:
I wonder if this parsing needs to be it's own step so that we can preserve the pVACseq CWL with alleles as an input for cases where we need to run this workflow directly (e.g., when we have clinical HLA typing information or want to run class I and class II alleles [which is not supported by optitype]).

Reply:
Changed in latest commit